axes/tests/test_access_attempt.py

```diff
@@ -394,3 +394,49 @@ def test_custom_authentication_backend(self):
         request.user = self.user
         authenticate(request=request, foo='bar')
         self.assertEqual(AccessLog.objects.all().count(), 0)
+
+    # by default, AXES_RESET_ON_SUCCESS = False
+    def test_reset_on_success_default(self):
```

Comment:
This is exactly the same implementation as the function above this. Refactor these into a single common function that is called from both test cases and not prefixed `test` so it doesn't get executed automatically. Something like `_assert_resets_on_success` or similar could be neat.

Reply:
I refactored the tests into a common function, I hope you like the way I did it. Let me know if I can do something in a better way.

Reply:
Looks good to me.